app/services/langfuse/tasks/score_monitor/actions.py

```
@@ -0,0 +1,273 @@
+            True if action executed successfully, False otherwise
+        """
+        try:
+            handler_class = self._action_handlers.get(ActionType(action_type))
+            if not handler_class:
+                logger.warning(f"Unknown action type: {action_type}")
+                return False
+
+            handler = handler_class(action_config)
+            return await handler.execute(call_sid, score, current_lead)
+
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)